qark/plugins/crypto/setting_secure_random_seed.py

```diff
@@ -0,0 +1,57 @@
+import os
+import logging
+
+import javalang
+
+from qark.scanner.plugin import BasePlugin
+from qark.issue import Severity, Issue
+
+log = logging.getLogger(__name__)
+
+
+class SeedWithSecureRandom(BasePlugin):
+
+    INSECURE_FUNCTIONS = ("setSeed", "generateSeed")
+
+    def __init__(self):
+
+        BasePlugin.__init__(self, category="crypto", name="Random number generator is seeded with SecureSeed",
+                            description=("Specifying a fixed seed will cause a predictable sequence of numbers. "
+                                         "This may be useful for testing, but not for secure use"))
+
+        self.severity = Severity.WARNING
+
+    def _imports_secure_seed(self, tree):
+        """Checks if a tree imports java.security.SecureRandom, and returns True if the import exists"""
+        imports = tree.filter(javalang.tree.Import)
```

Comment:
This should just be `tree.imports`. This data is already available and should save walking the entire tree.